odoo/tools/_vendor/sessions.py

```diff
@@ -26,7 +26,7 @@
 from time import time
 
 from werkzeug.datastructures import CallbackDict
-from werkzeug.posixemulation import rename
+from os import replace # python>=3.3
```

Comment:
Comment is not really useful, though it could be present in the commit message if you really want to note it.

Optionally, the import could rename `replace` to `rename` to avoid having to update line 201.

Reply:
yepp. python>3.7 is already required in

https://github.com/milahu/odoo/blob/97da1c2106d2a0e76dff1353e56b74d70db55eb2/odoo/__init__.py#L19

`os.rename` != `os.replace`: rename fails on windows when target exists

Reply:
> `os.rename` != `os.replace`: rename fails on windows when target exists

I didn't suggest importing `os.rename`, I suggested importing `os.replace` *as* `rename`, that way th rest of the code doesn't need to be touched.

Reply:
> that way th rest of the code doesn't need to be touched.

its one line, import-aliasing is confusing

odoo runs with my patch, so it seems that no other file does `import sessions.rename`

Reply:
> odoo runs with my patch, so it seems that no other file does import sessions.rename

Of course not, that file is a vendoring of Werkzeug's sessions support from when it was split out.